app.py

```diff
@@ -22,7 +22,7 @@ def predict():
     output = np.round(prediction[0], 2)

```

Comment:
GPT summary of [bca048](https://github.com/SampatSuthar/ml-demo/blob/e0f7317c2380011a88795578df7ea854860713b3/app.py#bca048bb925593525329c255555e16a0d9875d19) - [1fcf8a](https://github.com/SampatSuthar/ml-demo/blob/5a56a0cf9f19e28cf9772dc31682017656dd18bc/app.py#1fcf8aed8558b2618a97636c2f6d6c3587c2a6ee):
* Changed the OpenAI API key from a hardcoded string to a secret variable.
* Added a prompt and model for OpenAI completion.